spec/factories/groups.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :group do
+    company {}
+    users { [] }
```

Comment:
I think you can just not assign these like the other factory you have. These both seem like defaults
```suggestion
    company
    users
```

Reply:
> I think you can just not assign these like the other factory you have. These both seem like defaults

We have decided to leave this as is for now, for some reason when we change it to
```ruby
users
```
we get an error telling us `Trait not registered: "users"` whenever we try to create a `group`.  On the other hand, if we do
```ruby
users { }
```
we get and error saying `undefined method `each' for nil:NilClass` if we try to create a group where we don't explicitly set the `users` field, like this: `create(:group, display_name: search_term, company: company)`